app/web/features/communities/CommunityInfoPage.test.tsx

```diff
@@ -23,6 +27,9 @@ const listAdminsMock = service.communities.listAdmins as jest.MockedFunction<
 const getUserMock = service.user.getUser as jest.MockedFunction<
   typeof service.user.getUser
 >;
+const getLiteUsersMock = service.user.getLiteUsers as jest.MockedFunction<
+  typeof service.user.getLiteUsers
+>;
```

Comment:
You can take the `getUserMock` out here and in the beforeEach and it should still pass!